src/aks-preview/azext_aks_preview/custom.py

```diff
@@ -1214,6 +1237,15 @@ def aks_update(cmd,     # pylint: disable=too-many-statements,too-many-branches,
         instance.api_server_access_profile = \
             _populate_api_server_access_profile(api_server_authorized_ip_ranges, instance)
 
+    if update_aad_profile:
+        if instance.aad_profile is None or not instance.aad_profile.managed:
+            raise CLIError('Cannot specify "--update-aad-tenant-id/--update-aad-admin-group-object-ids"'
```

Comment:
remove `update` to align with params names

Reply:
nice, fixed